src/windows/winapi.rs

```diff
@@ -128,17 +128,19 @@ impl<'a> Iterator for DeviceInfoSetIter<'a> {
 
     fn next(&mut self) -> Option<Self::Item> {
         let mut device_interface_data =
-            unsafe { mem::uninitialized::<setupapi::SP_DEVICE_INTERFACE_DATA>() };
-        device_interface_data.cbSize =
-            mem::size_of::<setupapi::SP_DEVICE_INTERFACE_DATA>() as minwindef::UINT;
+            mem::MaybeUninit::<setupapi::SP_DEVICE_INTERFACE_DATA>::zeroed();
```

Comment:
Note this change is actually a real bugfix. AFAICT, `mem::uninitialized` tried to use compiler magic to avoid crashes here, but it was never defined. Now I'm following the guidance of https://crates.io/crates/winapi to use `::zeroed`